web/static/js/main.js

```diff
@@ -1,758 +1,902 @@
+    // var html_scan_methods;
+    var j;
+    var k;
+
+    var HTMLData;
+    var target;
+    var module_name;
+    var target_event;
+    var options;
+    var date;
+    var html_options;
+    var html_target_event;
+    var html_module_name;
+    var html_date;
+
+    
+
+    for (i = 0; i < res.length; i++) {
+      console.log(res[i])
+      target = res[i]["target"];
+      //target_event = res[i]["info"]["event"];
+      options = res[i]["info"]["options"];
+      //date = res[i]["info"]["date"];
+      module_name = res[i]["info"]["module_name"]
+
+      // open_ports = res[i]["info"]["open_ports"];
+      // scan_methods = res[i]["info"]["scan_methods"];
+      // category = res[i]["info"]["category"];
+      
+      // html_categories = "";
+      // html_scan_methods = "";
+      // html_open_ports = "";
+      // html_description = "";
+      html_target_event = "";
+      html_options = "";
+      html_date = "";
+      html_module_name = "";
+
+      // for (j = 0; j < open_ports.length; j++) {
+      //   html_open_ports +=
+      //     "<p class='mb-1 bold label label-warning'>open_port:" +
+      //     open_ports[j] +
+      //     "</p> ";
+      //   if (j == 10) {
+      //     html_open_ports +=
+      //       "<p class='mb-1 bold label label-warning'>open_port: click to see more.</p> ";
+      //     break;
+      //   }
+      // }
+      // for (j = 0; j < category.length; j++) {
+      //   html_categories +=
+      //     "<p class='mb-1 bold label label-info'>category:" +
+      //     category[j] +
+      //     "</p> ";
+      //   if (j == 10) {
+      //     html_categories +=
+      //       "<p class='mb-1 bold label label-info'>category: click to see more.</p> ";
+      //     break;
+      //   }
+      // }
+      for (j = 0; j < module_name.length; j++) {
+          html_module_name +=
+            "<p class='mb-1 bold label label-info'>selected_modules:" +
+            module_name[j] +
+            "</p> ";
+          if (j == 10) {
+            html_module_name +=
+              "<p class='mb-1 bold label label-info'>selected_modules: click to see more.</p> ";
+            break;
+          }
         }
-    });
 
-    $("#next_btn").click(function () {
-        result_page = result_page + 1;
-        if (result_page == 1) {
-            $('#previous_btn').hide();
+      
+      // html_scan_methods = "";
+      // for (j = 0; j < scan_methods.length; j++) {
+      //   html_scan_methods +=
+      //     "<p class='mb-1 bold label label-primary'>selected_modules:" +
+      //     scan_methods[j] +
+      //     "</p> ";
+      //   if (j == 10) {
+      //     html_scan_methods +=
+      //       "<p class='mb-1 bold label label-primary'>selected_modules: click to see more.</p> ";
+      //     break;
+      //   }
+      // }
+      //console.log(options)
+      var profile_array = []
+        //options[0]["profiles"]: ["medium severity", "low severtiy"]
+      profile_array = options[0]["profiles"] 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/OWASP/Nettacker/pullRequest?prid=7974804)
```suggestion
      profile_array = options[0]["profiles"]; 
```

Reply:
@hblankenship  Hi Harold, is this comment issued by codacy? Please review the integration from our repo since we can't manage it.